Add section outlining configuration of AUTH_OIDC_CACHE_ENTRY_SIZE (#10)
ni/systemlink-operations-handbook: handbook/openid-connect/openid-connect.md

    SystemLink uses log rotation therefore the latest logs may be in one of the numbered `error.log` files.

**Returned Claims:** See [**Viewing Claims Returned by a Provider**](#viewing-claims-returned-by-a-provider).

### Cache Entry Size

> The OpenID Connect module stores information in a shared memory cache. If a cache entry is too large, users will see an "Internal Server Error" when attempting to log in, and the NI Web Server logs will contain the following entries

Reviewer: Can we add a mention that you can get into this state if you are returning a large number of claims?

Do we know of any other potential ways we could get into this state?

Reviewer 2: Or claims with large values

```
    oidc_cache_shm_set: could not store value since value size is too large
    oidc_cache_set: could NOT store X bytes in shm cache backend for key Y

> ```

Reviewer 3: Until we get it added to the build workflow, please open the Markdown file in VS Code with the markdownlint extension and resolve any warnings:

- There should be blank lines before and after code snippets.
- You should specify the language for the code snippet (text).
- For consistency, we should wrap it in an admonition.

How about something like:

```md
The OpenID Connect module stores information in a shared memory cache. If a cache entry is too large, users will see an **Internal Server Error** when attempting to log in.

!!! note "Example Error Logs"
    When this happens, the NI Web Server error logs will contain entries like the following:

    ```text
    oidc_cache_shm_set: could not store value since value size is too large
    oidc_cache_set: could NOT store X bytes in shm cache backend for key Y
    ```
```

> To resolve this issue

Reviewer 3: Should this end with a colon?
```suggestion
To resolve this issue:
```

- Open the following file in a text editor run as Administrator `C:\Program Files\National Instruments\Shared\Web Server\conf\defines.d\50_mod_auth_openidc-defines.conf`
- Find the line `Define AUTH_OIDC_CACHE_ENTRY_SIZE 66065`

> - Modify `66065` to a number larger than X, where X is the required size of the cache entry specified in the error log

Reviewer 3: Instead of specifying the exact number, should be say to "find the line starting with ..." and to change the number at the end?

    SystemLink uses log rotation therefore the latest logs may be in one of the numbered `error.log` files.

**Returned Claims:** See [**Viewing Claims Returned by a Provider**](#viewing-claims-returned-by-a-provider).

> ### Cache Shm Entry Size Max

Reviewer 2: ```suggestion
### Cache Entry Size
```

> If an "Internal Server Error" is encountered when attempting to log in, this may be an issue with the defined `OIDCCacheShmEntrySizeMax`. You can verify this is the issue by searching the NI Web Server logs for the following entries

Reviewer 2: Might be good to have a little higher-level description here, something along the lines of:
```suggestion
The OpenID Connect module stores information in a shared memory cache. If the cache entry is too small, users will see an internal server error when attempting to log in, and the NI Web Server logs will contain the following entries:
```

```
    oidc_cache_shm_set: could not store value since value size is too large
    oidc_cache_set: could NOT store X bytes in shm cache backend for key Y
```
To resolve this issue

> - Open the following file in a text editor run as Administrator `C:\Program Files\National Instruments\Shared\Web Server\conf\defines.d\50_mod_auth_openidc-defines.conf.defaults`

Reviewer 2: 50_mod_auth_openidc-defines.conf.defaults is used to create 50_mod_auth_openidc-defines.conf on install, and to merge in new settings on upgrade. The server uses the configuration in 50_mod_auth_openidc-defines.conf.

```suggestion
- Open the following file in a text editor run as Administrator `C:\Program Files\National Instruments\Shared\Web Server\conf\defines.d\50_mod_auth_openidc-defines.conf`
```

- Find the line `Define AUTH_OIDC_CACHE_ENTRY_SIZE 66065`
- Modify `66065` to a number larger than X, where X is the required size of the shm cache entry specified in the error log

>     - It is suggested to follow the [documentation](https://github.com/zmartzone/mod_auth_openidc/blob/master/auth_openidc.conf#L600https://github.com/zmartzone/mod_auth_openidc/blob/master/auth_openidc.conf#L600) when modifying this number

Reviewer 2: I don't think we need to link to the external documentation, the content you're adding here gives the user what they need.

    SystemLink uses log rotation therefore the latest logs may be in one of the numbered `error.log` files.

**Returned Claims:** See [**Viewing Claims Returned by a Provider**](#viewing-claims-returned-by-a-provider).

### Cache Entry Size

The OpenID Connect module stores information in a shared memory cache. If a cache entry is too large, users will see an "Internal Server Error" when attempting to log in. This typically occurs when you are returning a large number of claims or claims with large values.

!!! note "Example error logs"
    When this happens, the NI Web Server error logs will contain entries like the following:

    ```text
    oidc_cache_shm_set: could not store value since value size is too large
    oidc_cache_set: could NOT store X bytes in shm cache backend for key Y
    ```

To resolve this issue:

> - Open the following file in a text editor run as Administrator `C:\Program Files\National Instruments\Shared\Web Server\conf\defines.d\50_mod_auth_openidc-defines.conf`

Reviewer 4: 'Administrator:'

(since it's introducing the file path)

- Find the line starting with `Define AUTH_OIDC_CACHE_ENTRY_SIZE`
- Modify the number at the end to a number larger than X, where X is the required size of the cache entry specified in the error log

> - Restart the NI Web Server from the `Control` tab of the NI Web Server Configuration application

Reviewer 4: Consider numbering these steps -- unordered lists can seem like just a list of options
